lib/backend/dbi.c

```diff
@@ -59,6 +64,15 @@ dbDetectBackend(rpmdb rdb)
     }
 #endif
 
+#if defined(WITH_SQLITE)
+    path = rstrscat(NULL, dbhome, "/Packages.sqlite", NULL);
```

Comment:
For this, we only have the single database file, right? Couldn't we just call this `rpmdb.sqlite` instead of `Packages.sqlite`?

Reply:
The thought has crossed my mind as well. The "reason" to have it resemble the others is basically to allow covering the different variants with "Packages*" but then LMDB doesn't follow that and we'll need an API to cover for the differences anyway so yeah, we could just as well call it something more sensible.

In general, naming (including database columns) is pretty much in the open still.